FinanceSchedulerDemo/SpecflowTests/Pages/LoginPage.cs

```diff
@@ -0,0 +1,44 @@
+﻿using OpenQA.Selenium;
+using System.Threading;
+
+namespace SpecflowTests.Pages
+{
+    public class LoginPage
+    {
+        public IWebDriver webDriver { get; }
+
+        public LoginPage(IWebDriver webDriver)
+        {
+            this.webDriver = webDriver;
+        }
+
+        public IWebElement LoginTab => webDriver.FindElement(By.Id("loginTab"));
+        public IWebElement UserNameInput => webDriver.FindElement(By.Id("userName"));
+        public IWebElement PasswordInput => webDriver.FindElement(By.Id("password"));
+        public IWebElement LoginBtn => webDriver.FindElement(By.Id("loginButton"));
+        public IWebElement UserAccountInfoCard => webDriver.FindElement(By.Id("userInfoCard"));
+
+        public void ClickLoginTab()
+        {
+            LoginTab.Click();
+        }
+
+        public void EnterCredentials(string userName, string password)
+        {
+            UserNameInput.SendKeys(userName);
+            PasswordInput.SendKeys(password);
+        }
+
+        public void ClickLoginButton()
+        {
+            Thread.Sleep(1000); // change with selenium wait
```

Comment:
Use fluent wait instead of thread sleep.
It should look like LoginBtn.WaitUntilClickable or Helper.WaitUntilBtnIsClickable